### PR TITLE
chore: add modality document

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

File: docs/tutorials/multimodality.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Multimodality in AI DIAL
+
+AI DIAL taps into this by connecting to Large Language Models (LLMs) that handle various media types. You can create applications to handle specific modality tasks or even comprehensive solution (orchestrators) to blend together applications for more complex scenarios.
+
+Working with different types of media is made available by supporting working with [files](https://epam-rail.com/dial_api#tag/Files). User or application in AI DIAL, can input and output files that are saved in a dedicated bucket and are accessible based on a flexible permissions model. Files can be provided as an input for multi-modality models and generated by them as an output.
+
+### Models
+
+[AI DIAL Chat application](../user-guide#language-models) offers user interface for communication with all the [Supported Models](../supported-models).
```
Comment (adubovik) [inline docs/tutorials/multimodality.md:15]: `all` is redundant

File: docs/tutorials/multimodality.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* For image-to-text tasks, AI DIAL has adapters for GPT4-Vision, Claude 3 and Gemini Pro Vision models.
+* For text-to-image tasks, AI DIAL has adapters for DALL-E-3, Google Imagen and Stability diffusion models.
+
+For audio/video-to-text tasks, AI DIAL has adapters for Gemini 1.5 Pro and Flash and Gemini 1.0 Pro Vision. Refer to [Vertex Adapter](https://github.com/epam/ai-dial-adapter-vertexai/) to view all supported models.
+
+### Applications
+
+You can use [DIAL SDK](https://github.com/epam/ai-dial-sdk) to create custom applications, which are basically any custom logic with a conversation interface packaged as a ready-to-use solution. Refer to [Tutorials](../tutorials/quick-start-with-application) to learn how to create a simple application or watch a [demo video](../video%20demos/demos-for-developers/dial-develop-and-deploy).
+
+Such application can be designed and configured to use multi-modal LLMs to perform specific tasks or even form an ecosystem of applications that can interact with each other.
```
Comment (adubovik) [inline docs/tutorials/multimodality.md:32]: multimodal

File: docs/tutorials/multimodality.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+For audio/video-to-text tasks, AI DIAL has adapters for Gemini 1.5 Pro and Flash and Gemini 1.0 Pro Vision. Refer to [Vertex Adapter](https://github.com/epam/ai-dial-adapter-vertexai/) to view all supported models.
+
+### Applications
+
+You can use [DIAL SDK](https://github.com/epam/ai-dial-sdk) to create custom applications, which are basically any custom logic with a conversation interface packaged as a ready-to-use solution. Refer to [Tutorials](../tutorials/quick-start-with-application) to learn how to create a simple application or watch a [demo video](../video%20demos/demos-for-developers/dial-develop-and-deploy).
+
+Such application can be designed and configured to use multi-modal LLMs to perform specific tasks or even form an ecosystem of applications that can interact with each other. 
+
+In the Cookbook section, you can view several examples:
```
Comment (adubovik) [inline docs/tutorials/multimodality.md:34]: view => find

File: docs/tutorials/multimodality.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+* [Text-to-text](../Cookbook/dial-cookbook/examples/how_to_call_text_to_text_applications)
+* [Text-to-image](../Cookbook/dial-cookbook/examples/how_to_call_text_to_image_applications)
+* [Image-to-text](../Cookbook/dial-cookbook/examples/how_to_call_image_to_text_applications)
+
+### Orchestrator
+
+Besides applications, you can create a comprehensive solution that can combine several applications and models to perform more complex and interconnected tasks. Such solutions can be called orchestrators.
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md (hunk 3 above)
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md (hunk 2 above)
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Working with different types of media is made available by supporting working with [files](https://epam-rail.com/dial_api#tag/Files). User or application in AI DIAL, can input and output files that are saved in a dedicated bucket and are accessible based on a flexible permissions model. Files can be provided as an input for multi-modality models and generated by them as an output.
+
+### Models
+
+[AI DIAL Chat application](../user-guide#language-models) offers user interface for communication with all the [Supported Models](../supported-models).
+
+Connection to LLMs is realized using so-called **adapters**. Refer to [OpenAI](https://github.com/epam/ai-dial-adapter-openai), [Bedrock](https://github.com/epam/ai-dial-adapter-bedrock/?tab=readme-ov-file#supported-models), [Vertex](https://github.com/epam/ai-dial-adapter-vertexai/?tab=readme-ov-file#supported-models) adapters to view more details about them and the supported models. **Note**, that you can use [DIAL SDK](https://github.com/epam/ai-dial-sdk) to create custom model adapters.
+
+AI DIAL has adapters to all popular text-to-text processing LLMs. Refer to [Supported Models](../supported-models) to learn more.
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+Working with different types of media is made available by supporting working with [files](https://epam-rail.com/dial_api#tag/Files). User or application in AI DIAL, can input and output files that are saved in a dedicated bucket and are accessible based on a flexible permissions model. Files can be provided as an input for multi-modality models and generated by them as an output.
+
+### Models
+
+[AI DIAL Chat application](../user-guide#language-models) offers user interface for communication with all the [Supported Models](../supported-models).
+
+Connection to LLMs is realized using so-called **adapters**. Refer to [OpenAI](https://github.com/epam/ai-dial-adapter-openai), [Bedrock](https://github.com/epam/ai-dial-adapter-bedrock/?tab=readme-ov-file#supported-models), [Vertex](https://github.com/epam/ai-dial-adapter-vertexai/?tab=readme-ov-file#supported-models) adapters to view more details about them and the supported models. **Note**, that you can use [DIAL SDK](https://github.com/epam/ai-dial-sdk) to create custom model adapters.
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md (hunk 1 above)
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## Introduction
+
+Multimodality refers to the use of multiple modes or methods to convey information such as text, images, audio, and video, to create a more effective and engaging way of transmitting information.
+
+## Multimodality in AI DIAL
+
+AI DIAL taps into this by connecting to Large Language Models (LLMs) that handle various media types. You can create applications to handle specific modality tasks or even comprehensive solution (orchestrators) to blend together applications for more complex scenarios.
+
+Working with different types of media is made available by supporting working with [files](https://epam-rail.com/dial_api#tag/Files). User or application in AI DIAL, can input and output files that are saved in a dedicated bucket and are accessible based on a flexible permissions model. Files can be provided as an input for multi-modality models and generated by them as an output.
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply docs/tutorials/multimodality.md]: fixed

File: docs/tutorials/multimodality.md [hunk 8]
```diff
@@ -8,15 +8,15 @@ Multimodality refers to the use of multiple modes or methods to convey informati
...
+Connection to LLMs is realized using so-called **adapters**. Refer to [OpenAI](https://github.com/epam/ai-dial-adapter-openai), [Bedrock](https://github.com/epam/ai-dial-adapter-bedrock/?tab=readme-ov-file#supported-models), [Vertex](https://github.com/epam/ai-dial-adapter-vertexai/?tab=readme-ov-file#supported-models) adapters to learn more about them and the supported models. You can use [DIAL SDK](https://github.com/epam/ai-dial-sdk) to create custom model adapters.
 
-AI DIAL has adapters to all popular text-to-text processing LLMs. Refer to [Supported Models](../supported-models) to learn more. 
+AI DIAL has adapters to a variety of text-to-text processing LLMs. Refer to [Supported Models](../supported-models) to view th list of supported models.
```
Comment (adubovik) [inline docs/tutorials/multimodality.md:19]: th => the

File: docs/tutorials/multimodality.md [hunk 9]
```diff
@@ -29,17 +29,17 @@ For audio/video-to-text tasks, AI DIAL has adapters for Gemini 1.5 Pro and Flash
...
 
 ### Orchestrator
 
-Besides applications, you can create a comprehensive solution that can combine several applications and models to perform more complex and interconnected tasks. Such solutions can be called orchestrators. 
+Besides creating an application which solves a specific multimodal task, you could create a generic application which is aware of multimodal DIAL models and is able to use them as tools to solve a given task. We call such generic applications orchestrators. 
 
 [DIAL ChatHub](../video%20demos/demos/dial-chathub) is an example of a flow orchestrator that combines several applications and models into one unified access point. ChatHub can automatically route prompts to one of several agents (text-to-text applications, text-to-image applications, vision-to-text applications) depending on the task that needs to be performed. For example, if a user asks about weather, the Web RAG agent is engaged, if a user wants to output an image based on the text input - a specific application handles this task that is connected with a corresponding model. All this is done while interacting with one ChatHub solution.
```
Comment (adubovik) [inline docs/tutorials/multimodality.md:44]: What does "flow" mean in "flow orchestrator"?
It looks like a word salad if no clear definition is provided.

**Review by adubovik — APPROVED** [review]

**Review by adubovik — APPROVED** [review]

**Pull request merged**